src/InvestJob.jsx

```diff
@@ -143,30 +129,32 @@ export class InvestJob extends React.Component {
       {sessionID: value});
   }
 
-  loadState(sessionFilename) {
+  async loadState(sessionFilename) {
     /** Set this component's state to the object parsed from a JSON file.
     *
     * @params {string} sessionFilename - path to a JSON file.
     */
 
-    // const filename = path.join(CACHE_DIR, sessionFilename);
     if (fs.existsSync(sessionFilename)) {
       const loadedState = JSON.parse(fs.readFileSync(sessionFilename, 'utf8'));
+      
+      // Right now we saveState is only called w/in investExecute and only
+      // after invest has created a logfile, which means an invest logfile
+      // should always exist and can be used to get args values and initialize SetupTab.
+      const datastack = await fetchDatastackFromFile(

```

Comment:
If the user accidentally deletes the log file or moves it around what happens to this call?

Reply:
Great question. I'm adding some tests to make sure this is handled well.

Reply:
I added a check here to see if `datastack` exists and raise an alert message if not. But this also brings up the larger issue of designing how application state and sessions are stored on disk. All of that functionality is still kind of a prototype. I've created issue #22 .